lib/cucumber/formatter/url_reporter.rb

```diff
@@ -0,0 +1,24 @@
+require 'cucumber/term/banner'
+
+module Cucumber
+  module Formatter
+    class URLReporter
+      include Term::Banner
+
+      def initialize(io)
+        @io = io
+      end
+
+      def report(url)
+        uri = URI(url)
+        display_banner(
+          [
+            'View your Cucumber Report at:',
+            [["https://reports.cucumber.io#{uri.path}", :blue, :underline]]
```

Comment:
Cucumber-JVM's `NoPublishFormatter` uses `:cyan` + `:bold` (and no underline). Some terminals (iTerm2) will underline URLs automatically, but maybe we should use underline in Cucumber-JVM too?

So agree on `:cyan` + `:bold` + `:underline`?

Reply:
I've updated Cucumber-JVM to do `:cyan` + `:bold` + `:underline` now

Reply:
done too